xdsl/backend/csl/print_csl.py

```diff
@@ -451,7 +451,7 @@ def print_block(self, body: Block):
                     if init is None:
                         init = ""
                     else:
-                        init = f" = { self.attribute_value_to_str(init)}"
+                        init = f" = { self._var_use(init)}"
```

Comment:
`_var_use` potentially initialises the variable, I think `_get_variable_name_for` would be better here.

(strictly speaking, I don't think there is any way this can generate `const foo = const bar`, since `init` is an operand, but it could in theory?)

Reply:
I actually have a case where that happens, will fix!